docs/concepts.md

```diff
@@ -0,0 +1,94 @@
+<!--[metadata]>
++++
+title = "Machine concepts and help"
+description = "Understand concepts for Docker Machine, including drivers, base OS, IP addresses, environment variables"
+keywords = ["docker, machine, amazonec2, azure, digitalocean, google, openstack, rackspace, softlayer, virtualbox, vmwarefusion, vmwarevcloudair, vmwarevsphere, exoscale"]
+[menu.main]
+parent="workw_machine"
+weight=3
++++
+<![end-metadata]-->
+
+
+# Understand Machine concepts and get help
+
+Docker Machine allows you to provision Docker on virtual machines that reside
+either on your local system or on a cloud provider. Docker Machine creates a
+host on a VM and you use the Docker Engine client as needed to build images and
+create containers on the host.
+
+## Drivers for creating virtual machines
+
+To create a virtual machine, you supply Docker Machine with the name of the
+driver you want use. The driver represents the virtual environment. For example,
```

Comment:
Hi Chanwit,
I believe I incorporated all your comments. Please take a look at the
updated Concepts topic on GitHub for this PR,
<https://github.com/docker/machine/pull/2957#issuecomment-177574034> and/or
at Machine concepts and help <http://104.131.118.86:8000/machine/concepts/>
on the draft docs site.

Thanks for the comments--all good catches. --Vicky

On Sun, Jan 31, 2016 at 11:55 AM, Chanwit Kaewkasi <notifications@github.com
> wrote:

> In docs/concepts.md
> <https://github.com/docker/machine/pull/2957#discussion_r51366143>:
>
> > +weight=3
> > ++++
> > +<![end-metadata]-->
> > +
> > +
> > +# Understand Machine concepts and get help
> > +
> > +Docker Machine allows you to provision Docker on virtual machines that reside
> > +either on your local system or on a cloud provider. Docker Machine creates a
> > +host on a VM and you use the Docker Engine client as needed to build images and
> > +create containers on the host.
> > +
> > +## Drivers for creating virtual machines
> > +
> > +To create a virtual machine, you supply Docker Machine with the name of the
> > +driver you want use. The driver represents the virtual environment. For example,
>
> The driver represents the virtual environment. For example, on a local
> Linux, Mac, or Windows system the driver is typically Oracle VirtualBox.
>
> This statement is also not correct for generic and SoftLayer's drivers.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/machine/pull/2957/files#r51366143>.
>